packages/dx-react-demos/src/demo-sources/grid-column-groups/basic.jsxt

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';<%&additionalImports%>
+import {
+  Grid,
+  Table,
+  TableGroupHeader,
+  TableHeaderRow,
+} from '@devexpress/dx-react-grid-<%&themeName%>';
+import { countries } from '../../../demo-data/countries';
+
+const PercentCell = ({ value, ...restProps }) => (
```

Comment:
Why don't you use `DataTypeProvider` for cell value formatting?

Reply:
Because we should show a minimal functionality. The `DataTypeProvider` is a new dependency in this demo.